service/frontend/workflowHandler.go

```diff
@@ -3512,6 +3512,17 @@ func (wh *WorkflowHandler) ReapplyEvents(
 	return nil
 }
 
+// GetCadenceDeploymentInfo return information about cadence deployment
+func (wh *WorkflowHandler) GetCadenceDeploymentInfo(ctx context.Context) (resp *gen.DeploymentInfo, err error) {
```

Comment:
Right now I am not checking for any error. Any suggestion on errors that needs to be handled?

Reply:
Ideally this call should return some flag that indicated that cluster is healthy and ready to take traffic. It is needed for example during docker startup to wait for service to start by polling this function.

Reply:
health check API and cluster info API should be 2 different APIs

Reply:
i would also suggest making this API `GetClusterInfo`